pkg/printers/humanreadable.go

```diff
@@ -279,6 +281,11 @@ func printHeader(columnNames []string, w io.Writer) error {
 
 // PrintObj prints the obj in a human-friendly format according to the type of the obj.
 func (h *HumanReadablePrinter) PrintObj(obj runtime.Object, output io.Writer) error {
+	return h.PrintObjEvent("", obj, output)
+}
+
+// PrintObj prints the obj in a human-friendly format according to the type of the obj.
+func (h *HumanReadablePrinter) PrintObjEvent(eventType string, obj runtime.Object, output io.Writer) error {
```

Comment:
from https://github.com/kubernetes/kubernetes/pull/72418#discussion_r244656712:

> @smarterclayton:
> Why not just pass Event to PrintObj and have it to do the right thing? This feels like a workaround to the underlying structure, and events are runtime.Objects just like everything else.

Reply:
Hrmm... I wasn't excited about teaching all the printers about unwrapping Event objects, but maybe that would be better.

Reply:
well... it's different... your call if it's better